CHANGELOG.md

```diff
@@ -6,18 +6,29 @@
 
 ### feat(frontend-canister): add `get_asset_properties` and `set_asset_properties` to frontend canister
 
-### feat: write canister metadata sections for dfx pull
+It's now possible to get and set AssetProperties for assets in frontend canister, like so:
+```
+dfx canister call e2e_project_frontend set_asset_properties '( record { key="asset.txt"; max_age=opt(opt(5:nat64)); headers=opt(opt(vec{record {"new-key"; "new-value"}})) })'
+dfx canister call e2e_project_frontend get_asset_properties '("/asset.txt")'
+```
+If you wish to set either of properties to `null`, do the following:
+```
+dfx canister call e2e_project_frontend set_asset_properties '( record { key="asset.txt"; max_age=opt(null); headers=opt(null) })'
```

Comment:
Is this how we want to describe the usage in the changelog?  Wouldn't developers instead configure this through ic-asset.json?  Describing `canister call` as the method for setting this data will be confusing, in opinion.   Given that the sync process doesn't yet handle updates of this type of data, I would suggest to mention that these fields can be set in ic-asset.json but will only affect newly created files, until we implement proper synchronization handling.

In other words, at this point this is more like "support for future work."

Reply:
right, I like this better

I guess I wanted to empower our users, but you may be right that it could cause some confusion